lib/rubocop/cop/chef/style/use_platform_helpers.rb

```diff
@@ -18,23 +18,27 @@ module RuboCop
   module Cop
     module Chef
       module ChefStyle
-        # Use the platform?() and platform_family?() helpers instead of node['platform] == 'foo'
-        # and node['platform_family'] == 'bar'. These helpers are easier to read and can accept multiple
-        # platform arguments, which greatly simplifies complex platform logic.
+        # Use the platform?() and platform_family?() helpers instead of node['platform] == 'foo' and node['platform_family'] == 'bar'. These helpers are easier to read and can accept multiple platform arguments, which greatly simplifies complex platform logic.
```

Comment:
nit: a better multi-line comment?

Reply:
The multi line comments result in poorly formatted docs pages. The single line comments just wrap in the HTML and do the right thing. Once I started generating the docs I started slowly removing the multi-line comments.